mobx_codegen/lib/src/errors.dart

```diff
@@ -152,6 +156,16 @@ class InvalidObservableAnnotations extends PropertyErrors {
       'Remove @observable annotation for $property $propertyList. They only apply to fields.';
 }
 
+class InvalidReadOnlyAnnotations extends PropertyErrors {
+  @override
+  // ignore: overridden_fields
+  Pluralize propertyPlural = _fieldPluralizer;
+
+  @override
+  String get message =>
+      'Remove @readonly annotation from $property $propertyList. They only apply to private fields.';
```

Comment:
Please check this message. Is private ==> public ?

Reply:
So you want to change the message? No problem. But what's the new one? 'Is private ==> public'?

Reply:
No I mean, is this for private fields or public? Perhaps make it more clear

Reply:
Oh I see. The annotation @readonly should be used only on private fields to prevent public setters.

Reply:
I used this message because it was similar to others.